IdentityCore/src/validation/MSIDAADAuthority.m

```diff
@@ -234,6 +234,16 @@ - (BOOL)supportsMAMScenarios
     return YES;
 }
 
+- (BOOL)checkTokenEndpointForRTRefresh:(NSURL *)tokenEndpoint
+{
+    NSArray *environmentAliases = self.defaultCacheEnvironmentAliases;
+    
+    if (!tokenEndpoint || ![environmentAliases count])
+        return YES;
+    
+    return [tokenEndpoint.host.msidNormalizedString msidIsEquivalentWithAnyAlias:environmentAliases];
```

Comment:
Do we need to worry about upper/lower case here?

Reply:
`msidIsEquivalentWithAnyAlias` already takes care of that :)